src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -358,34 +358,37 @@ public function makePathRelative($endPath, $startPath)
             $startPath = str_replace('\\', '/', $startPath);
         }
 
+        $stripDriveLetter = function ($path) {
+            if (strlen($path) > 2 && substr($path, 1, 2) === ':/' && ctype_alpha($path[0])) {
```

Comment:
Should I change the `substr()` to `':' === $path[1] && '/' === $path[2]` to get this pull request approved?